common/eth2/src/lighthouse/sync_committee_rewards.rs

```diff
@@ -0,0 +1,23 @@
+use serde::{Deserialize, Serialize};
+
+// Details about the rewards paid to sync committee members for attesting headers
+// All rewards in GWei
+
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize)]
+pub struct SyncCommitteeRewards {
+
+    pub execution_optimistic: Option<bool>,
+
+    pub finalized: Option<bool>,
```

Comment:
Please remove these fields and use `GenericResponse` + `add_execution_optimistic`.

For the `finalized` field we can just leave it blank until #3753 is merged.